lapce-app/src/command.rs

```diff
@@ -183,6 +183,10 @@ pub enum LapceWorkbenchCommand {
     #[strum(message = "Open File")]
     OpenFile,
 
+    #[strum(serialize = "select_in_file_explorer")]
+    #[strum(message = "Select in File Explorer")]
+    SelectInFileExplorer,
```

Comment:
It should be named `RevealInFileTree` or `RevealActiveFileInFileTree`

Reply:
done